spec/helpers/notices_helper_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe NoticesHelper do
+  include NoticesHelper
+
+  context 'permanent_url_full_notice' do
```

Comment:
What should it do when a token URL exists, but is invalid (expired, wrong user, etc)?

Reply:
> wrong user

I've added a test for calling the method with a guest user.

> expired

This is a permanent url, it's never expired.

Reply:
👍